pkgs/dart_tooling_mcp_server/lib/src/mixins/dtd.dart

```diff
@@ -31,7 +32,7 @@ base mixin DartToolingDaemonSupport on LoggingSupport, ToolsSupport {
   /// [VmService] objects are automatically removed from the Map when the
   /// [VmService] shuts down.
   @visibleForTesting
-  final activeVmServices = <String, VmService>{};
+  final activeVmServices = <String, Future<VmService>>{};
```

Comment:
the dart doc needs to be updated since the key for this map changed (debug session id vs vm service uri).

Generally, why the change from vm service uri as the key to debug session id?

Reply:
When we get the event for a debug session ending, a VM service URI is not included, only the debug session ID